### ISSUE TITLE
Command to visualize selected societies on a map

### ISSUE THREAD

**Issue by xrotwang** [D-PLACE/dplace-data#150]
To make it easier to check locations of societies (e.g. to prevent issues like #149) it would be nice to be able to see societies (of a dataset) on a map. This could be implemented by putting together an HTML page - possibly with inlined GeoJSON - displaying a leaflet map, and opening this map in the browser.

**Comment by xrotwang** [comment]
Note: This kind of functionality could be reused by [pyglottolog](https://github.com/clld/glottolog), thus it should be implemented in a library, maybe `clldutils`.

**Comment by SimonGreenhill** [comment]
I agree. Given that @Bibiko was trying to make an offline savable map, rather than a leaflet map would it make sense to kill two birds with one stone and generate a png/pdf/svg/whatever?

**Comment by xrotwang** [comment]
Yes, possibly. Just came across [Kartograph](http://kartograph.org/docs/kartograph.py/) which may fit the bill.

**Comment by xrotwang** [comment]
Hm, Kartograph isn't maintained anymore and the former maintainer recommends [mapshaper](https://github.com/mbloch/mapshaper) - but mapshaper requires node.js ...

Anyway, if Kartograph works for us, this should be good enough.
